### ISSUE TITLE
[Feature Request] Integration with Autoforms and Collection2?

### ISSUE THREAD

**Issue by mukulhase** [perak/kitchen-site#390]
It would be great if you could use Autoforms and collection2 instead of the custom ui components, this would allow for many more types of forms and schemas which could still fit in the current meteor kitchen UI.

**Comment by perak** [comment]
I agree! BTW, you can already generate collection2 schema, see use_collection2 property of application object ( in API docs )

**Comment by paradoxewan** [comment]
+1 for this

**Comment by perak** [comment]
I know this is old issue but partly solved now:

- Collection2 - done. 
- Autoform possible (but not fully automatic)

(not closing issue yet)
